commit 1eb8dd51887bbcdd08220828f693105e9e3459fe
Author: Bjorn Andersson <bjorn.andersson@linaro.org>
Date:   Fri Nov 2 14:45:32 2018 -0700

    arm64: dts: sdm845-mtp: Reserve reserved gpios
    
    [ Upstream commit 5f8d3ab136d0ccb59c4d628d8f85e0d8f2761d07 ]
    
    With the introduction of commit 3edfb7bd76bd ("gpiolib: Show correct
    direction from the beginning") the gpiolib will attempt to read the
    direction of all pins, which triggers a read from protected register
    regions.
    
    The pins 0 through 3 and 81 through 84 are protected, so mark these as
    reserved.
    
    Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>
    Reviewed-by: Stephen Boyd <sboyd@kernel.org>
    Reviewed-by: Linus Walleij <linus.walleij@linaro.org>
    Signed-off-by: Andy Gross <andy.gross@linaro.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm64/boot/dts/qcom/sdm845-mtp.dts b/arch/arm64/boot/dts/qcom/sdm845-mtp.dts
index 6d651f314193..6921f8dc5ebb 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-mtp.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-mtp.dts
@@ -31,6 +31,10 @@
 	status = "okay";
 };
 
+&tlmm {
+	gpio-reserved-ranges = <0 4>, <81 4>;
+};
+
 &uart9 {
 	status = "okay";
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1033_1eb8dd51887bbcdd08220828f693105e9e3459fe.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1033_1eb8dd51887bbcdd08220828f693105e9e3459fe.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
